docs/adr/0051-audit-log-part-j.md

```
@@ -0,0 +1,107 @@
+- **Version:** v3.7.0
+- **PR:** [#1591](https://github.com/corazawaf/coraza/pull/1591)
+- **Issue(s):** No linked issue
+- **Deciders:** @fzipi, @M4tteoP, @coderabbitai
+- **Category:** Parity (ModSecurity v2 Part J parity)
+
+## Context and Problem
+
+ModSecurity v2 defines `Part J` of `SecAuditLogParts` as a list of
+uploaded-file metadata:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)